src/sign_workflow/sign_artifacts.py

```diff
@@ -23,14 +23,16 @@ class SignArtifacts:
     artifact_type: str
     signature_type: str
     platform: str
+    allow_output_override: str
     signer: Signer
 
-    def __init__(self, target: Path, components: List[str], artifact_type: str, signature_type: str, platform: str) -> None:
+    def __init__(self, target: Path, components: List[str], artifact_type: str, signature_type: str, platform: str, allow_output_override: str) -> None:
```

Comment:
Why is string used here for the arg value?
I would expect True/False to be bool, or yes/no to be str for the value.

Reply:
Because in the underlying client is again shell and then python. So it's Groovy -> shell -> python -> shell -> python end to end workflow during which even a 'False' boolean is considered as True

Reply:
Got it, thanks.